WordPress/Classes/ViewRelated/Menus/MenuItemAbstractPostsViewController.m

```diff
@@ -42,7 +44,13 @@ - (NSPredicate *)defaultFetchRequestPredicate
 
 - (void)syncPosts
 {
+    if (self.isSyncing) {
```

Comment:
I'm digging it, but fwiw, we also have a WPContentSyncHelper that encapsulates sync state wrangling.